### DISCUSSION TITLE
how to install on Mac os ?

### DISCUSSION THREAD

**Discussion by antlep** [pierotofy/OpenSplat#132] [Q&A]
Hi folks,

I tried to "build" the repo on my M2Pro Mac. Readme miss something because
> cmake -DCMAKE_PREFIX_PATH=/path/to/libtorch/ -DGPU_RUNTIME=MPS .. && make -j$(sysctl -n hw.logicalcpu)
./opensplat

gives this error :

`zsh: no such file or directory: ./opensplat`

**Comment by jared-krauss** [comment]
Hey, I used homebrew to install all of these, and I had to correctly specify where to locate libtorch. 

I am not good at commenting on the actual github, but I have a google drive doc exploring my attempts to get a GS made on my Mac. 

It's here if you want to follow it along for yourself, feel free to comment where you struggle or get stuck. 
https://docs.google.com/document/d/1Qfyar0fL89X2R9JqByIpli1LtgETBIXKnbNzIyfjHx4/edit?usp=sharing
